projects/plugins/jetpack/modules/search/customberg/components/sidebar/sidebar-options.jsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Panel,
+	PanelBody,
+	RadioControl,
+	SelectControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useSiteLoadingState from '../../hooks/use-loading-state';
+import useEntityRecordState from '../../hooks/use-entity-record-state';
+import useSearchOptions from '../../hooks/use-search-options';
+import ColorControl from './color-control';
+import ExcludedPostTypesControl from './excluded-post-types-control';
+import ThemeControl from './theme-control';
+
+/* eslint-disable react/jsx-no-bind */
+
+/**
+ * Customization/configuration tab for the sidebar.
+ *
+ * @returns {Element} component instance
+ */
+export default function SidebarOptions() {
+	const {
+		color,
+		excludedPostTypes,
+		infiniteScroll,
+		resultFormat,
+		setColor,
+		setExcludedPostTypes,
+		setInfiniteScroll,
+		setResultFormat,
+		setShowLogo,
+		setSort,
+		setSortEnabled,
+		setTheme,
+		setTrigger,
+		showLogo,
+		sort,
+		sortEnabled,
+		theme,
+		trigger,
+	} = useSearchOptions();
+
+	const { isSaving } = useEntityRecordState();
+	const { isLoading } = useSiteLoadingState();
+	const isDisabled = isSaving || isLoading;
+
+	return (
+		<Panel
+			className={ classNames( 'jp-search-customize-sidebar-options', {
+				'jp-search-customize-sidebar-options--is-disabled': isDisabled,
```

Comment:
`BEM` or Calypso standard is a question here. As @bluefuton suggested, we should follow [this](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md), which is similar but different to `BEM`. Modifiers would be just like `is-disabled` if I haven't got it wrong.

Reply:
Let's go with BEM as much as possible. I can imagine modifier classes like `is-disabled` wreaking havoc via style conflicts in the future.